source/content/guides/decoupled/drupal-backend-starters/07-cors-configuration.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Decoupled Drupal Backend Starter for Front-End Sites
+subtitle: CORS Configuration
+description: Learn how to configure CORS.
+tags: [webops, workflow, decoupled]
+contributors: [backlineint, cobypear, hckia, whitneymeredith]
+layout: guide
+permalink: docs/guides/decoupled/drupal-backend-starters/cors-configuration
+anchorid: cors-configuration
+contenttype: [guide]
+innav: [true]
+categories: [create]
+cms: [decoupled]
+audience: [development]
+product: [decoupled]
+integration: [--]
+---
+
+This section provides information on how to configure your cross-origin resource sharing (CORS).
```

Comment:
This page is nearly identical to https://decoupledkit.pantheon.io/docs/backend-starters/decoupled-drupal/cors-configuration

What's the long term plan for managing similar/identical docs across these two sites?

Reply:
@stevector All Front-End Site public content will be on the Docs site (expect for packages info and repo info). Over time the decoupledkit. pantheon.io will be phased out.

Reply:
@EdwardAngert do you have perspective here? Do you know what Joan intended?

Reply:
@backlineint I think both sites will continue to exist for the duration of EA. However, I was under the impression that when the product eventually reaches GA, that the decoupledkit. pantheon.io site would most likely be phased out. This might be a misunderstanding on my part. The Docs site content is for developer use. The intention is to provide a conceptual introduction to the product and the technical information and steps required to use the product. This is the same format that all our guides use.